src/components/ScreenWrapper/index.js

```diff
@@ -133,6 +133,18 @@ class ScreenWrapper extends React.Component {
                                     <OfflineIndicator />
                                 )}
                             </KeyboardAvoidingView>
+                            {Platform.OS === 'android' && Platform.Version < 29 && (
```

Comment:
Is it possible to use a platform-specific file for this, to avoid having android logic in the generic file